app/scripts/components/common/map/style-generators/raster-paint-layer.tsx

```diff
@@ -52,10 +52,34 @@ export function RasterPaintLayer(props: RasterPaintLayerProps) {
 
   useEffect(
     () => {
-      const tileParamsAsString = qs.stringify(updatedTileParams, {
+      // Create a modified version of the parameters
+      const processedParams = { ...updatedTileParams } as {
+        reScale?: number[];
+        colormap_name?: string;
+        bands?: string[];
```

Comment:
I can't find docs about the bands parameter (https://openveda.cloud/api/raster/docs#/STAC%20Search/tile_searches__search_id__tiles__tileMatrixSetId___z___x___y___scale_x__format__get) - am I missing anything?

Reply:
Found it! forgot that titiler cmr is a separate instance : https://dev-titiler-cmr.delta-backend.com/api.html#/Raster%20Tiles/tiles_endpoint_tiles__tileMatrixSetId___z___x___y___format__get
thank you @vincentsarago !